hack/aks/Makefile

```diff
@@ -234,23 +299,41 @@ swift-up: rg-up swift-net-up ## Bring up a SWIFT AzCNI cluster
 	@$(MAKE) set-kubeconf
 
 swiftv2-multitenancy-cluster-up: rg-up
+	$(AZCLI) network public-ip create --name $(PUBLIC_IP) \
+		--resource-group $(GROUP) \
+		--allocation-method Static \
+		--ip-tags $(IP_TAG) \
+		--location $(REGION) \
+		--sku Standard \
+		--tier Regional \
+		--version IPv4
```

Comment:
instead of copy/pasting these 8 lines in to every cluster target, creating the public IP should be it's own target that can be reused like the `set-kubeconf` target
```make
public-ip: rg-up
	$(AZCLI) network public-ip create --name $(PUBLIC_IP) \
		--resource-group $(GROUP) \
		--allocation-method Static \
		--ip-tags $(IP_TAG) \
		--location $(REGION) \
		--sku Standard \
		--tier Regional \
		--version IPv4

...
...

swiftv2-multitenancy-cluster-up: rg-up
	@$(MAKE) public-ip // <- calls the new public ip target
	$(AZCLI) aks create -n $(CLUSTER) -g $(GROUP) -l $(REGION) \
		--network-plugin azure \
		--network-plugin-mode overlay \
		--kubernetes-version 1.28 \
		--nodepool-name "mtapool" \
		--node-vm-size $(VM_SIZE) \
		--node-count 2 \
		--load-balancer-outbound-ips /subscriptions/$(SUB)/resourceGroups/$(GROUP)/providers/Microsoft.Network/publicIPAddresses/$(PUBLIC_IP) \
		--nodepool-tags fastpathenabled=true \
		--no-ssh-key \
		--yes
	@$(MAKE) set-kubeconf

...
```

Reply:
updated the PR to reflect this